fetch.bs

```diff
@@ -1892,6 +1892,24 @@ source of security bugs. Please seek security review for features that deal with
  <a for="URL serializer"><i>exclude fragment</i></a> set to true.
 </ol>
 
+<p>To check <dfn export>Cross-Origin-Embedder-Policy allows credentials</dfn>, given a
+<a for=/>request</a> <var>request</var>, run theses steps:
+
+<ol>
+ <li><p>If <var>request</var>'s <a for=request>mode</a> is not <code>no-cors</code>", return
```

Comment:
Missing quotation mark.

Reply:
then return* (throughout)

Reply:
Done:
> Missing quotation mark.

But, I have trouble understanding what means:
> then return* (throughout)

Reply:
Ah sorry I missed this, I meant that instead of "return", write "then return". That's the convention this specification uses for if statements.